doc/guides/source/plugin_table.textile

```diff
@@ -126,8 +134,16 @@ Aloha.settings.plugins: {
 		colResize: true,
 		// allow resizing the row height (options: "px", false, default: false)
 		rowResize: true,
-		//add a default class to newly created tables
-		defaultClass : 'table-responsive'
+		// add a class to newly-created tables (or multiple space-separated classes)
+		defaultClass: 'table-responsive'
+		// add a class to all rows
+		defaultRowClass: 'c-table__row',
+		// add a class to all header rows
+		defaultHeaderRowClass: 'c-table__row c-table__header',
+		// add a class to all cells
+		defaultCellClass: 'c-table__cell',
```

Comment:
When converting from cell to headerCell, remove the defaultCellClass and then add defaultHeaderCellClass and vice versa. Same for rows.